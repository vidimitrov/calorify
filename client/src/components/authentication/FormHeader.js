import styled from 'styled-components';

const FormHeader = styled.div`
  padding: 30px 0;
`;

export default FormHeader;
