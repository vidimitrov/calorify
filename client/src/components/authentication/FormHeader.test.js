import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FormHeader from './FormHeader';

it('Authentication/FormHeader component renders correctly', () => {
  const tree = renderer.create(<FormHeader>Some header</FormHeader>).toJSON();
  expect(tree).toMatchSnapshot();
});
