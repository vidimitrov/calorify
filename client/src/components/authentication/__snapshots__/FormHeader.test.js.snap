// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/FormHeader component renders correctly 1`] = `
.c0 {
  padding: 30px 0;
}

@media (max-width:420px) {
  .c0 {
    padding: 10px 0;
  }

  .c0 > h1 {
    margin: 0;
  }
}

<div
  className="c0"
>
  Some header
</div>
`;
